src/core/plugins/json-schema-2020-12-samples/fn/core/random.js

```diff
@@ -16,7 +16,9 @@ export const bytes = (length) => randomBytes(length)
 
 export const randexp = (pattern) => {
   try {
-    const randexpInstance = new RandExp(pattern)
+    const safeRegex = /(?<=\{[^}]*)\d{3,}(?=[^}]*\})/g
```

Comment:
`/(?<!\\)(?<=\{[^}]*)\d{3,}(?=[^}]*\}(?!\\))/g` - this should cover for the case of having the brackes escaped, which basically creates a stream of characters instead of control structure for regular expression.